apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,578 @@
+        level
+        minSalary
+        maxSalary
+      }
+      workStream {
+        name {
+          en
+          fr
+        }
+      }
+      language {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      department {
+        name {
+          en
+          fr
+        }
+      }
+      closingDate
+      opportunityLength {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      isRemote
+      location {
+        en
+        fr
+      }
+      securityClearance {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      areaOfSelection {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      processNumber
+      poolSkills {
+        id
+        type {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        skill {
+          id
+          key
+          name {
+            en
+            fr
+          }
+          category {
+            value
+            label {
+              en
+              fr
+            }
+          }
+        }
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationDialogProps {
+  applicationQuery: ReviewApplicationPreviewListFragment;
+}
+
+const ReviewApplicationDialog = ({
+  applicationQuery,
+}: ReviewApplicationDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const application = applicationQuery;
+  const pool = application?.pool;
+
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+  const poolName = getLocalizedName(pool?.name, intl);
+
+  // Separate essential and asset skills, sort them by category, and confirm they include skill data
+  const poolSkills = unpackMaybes(pool?.poolSkills);
+  const essentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Essential,
+    ),
+  );
+  const nonessentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Nonessential,
+    ),
+  );
+
+  const focusOnRecruitment = useRef(false);
+
+  const status = getApplicationStatusChip(
+    application.submittedAt,
+    pool.closingDate,
+    application.removedAt,
+    application.finalDecisionAt,
+    application.finalDecision?.value,
+    pool.areaOfSelection?.value,
+    application.assessmentStatus,
+    pool.screeningQuestions,
+    intl,
+  );
+
+  const isDraftStatus = status.value === ApplicationStatus.DRAFT;
+  const isExpiredStatus = status.value === ApplicationStatus.EXPIRED;
+  const lessThanThreeDaysTillClosingDate = pool?.closingDate
+    ? differenceInDays(Date.now(), parseDateTimeUtc(pool.closingDate)) < 3
+    : null;
+  const showDeadlineToApply =
+    isDraftStatus ||
```

Comment:
I don't think you want the first check of `isDraftStatus` here

Reply:
Fixed in [51cf495](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/51cf495516a1b728326d2427eda87c444d0702ed)